src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/IMsBuildAccessor.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Build.Evaluation;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Build
+{
+    /// <summary>
+    /// Utility class for allowing for testing of code that needs to access the msbuild lock, and also be testable.
+    /// </summary>
+    internal interface IMsBuildAccessor
```

Comment:
Yeah, I put it in Build because it is accessing msbuild. I'll find a better name/location.